bentoml/_internal/service/runner.py

```diff
@@ -1,24 +1,22 @@
+import abc
+
+from simple_di import Provide, inject
+from bentoml._internal.configuration.containers import BentoMLContainer
 import os
-import typing as t
-from abc import ABC, abstractmethod
+from typing import Optional, Union, Callable
 
 import attr
 import psutil
 
-from .utils import _cpu_converter, _mem_converter, _query_cpu_count
-
-if t.TYPE_CHECKING:  # pylint: disable=unused-import # pragma: no cover
-    import numpy as np
-    import pandas as pd
-
-_T = t.TypeVar("_T", bound=t.Union[t.List, "np.array", "pd.DataFrame"])
+from .utils import _cpu_converter, _mem_converter, _query_cgroup_cpu_count
 
 
 @attr.s
-class RunnerResourceLimits:
-    cpu = attr.ib(converter=_cpu_converter, type=float)
-    mem = attr.ib(converter=_mem_converter, type=int)
+class ResourceQuota:
+    cpu = attr.ib(converter=_cpu_converter, type=Optional[float])
+    ram = attr.ib(converter=_mem_converter, type=Optional[int])
```

Comment:
I think `memory` make more sense for users

Reply:
let's make it `memory` to keep it consistent with kubernetes' resource quota API